ship.config.js

```diff
@@ -1,17 +1,13 @@
 const shell = require('shelljs');
 
 const packages = JSON.parse(
-  shell.exec('yarn run --silent lerna list --toposort --json --no-private', {
+  shell.exec('yarn run --silent lerna list --toposort --json', {
     silent: true,
   })
 );
 
 module.exports = {
-  shouldPrepare: ({ releaseType, commitNumbersPerType }) => {
-    const { fix = 0 } = commitNumbersPerType;
-    if (releaseType === 'patch' && fix === 0) {
-      return false;
-    }
+  shouldPrepare: () => {
```

Comment:
Why always return `true`?

Reply:
Because the package returned from version possibly doesn't have any relevant pushes, while other packages would have stuff to release. Therefore I think it's best if we get the pr every two weeks and choose to close if it's irrelevant

Reply:
I think we need to document this process so we're all fully aware of it and can manage it. For now it's blurry to me how things would go and what are the different possible scenarios.

Also, we need to better handle this for when we have feature teams contribute, we can't be the only ones on duty of merging release PRs in the long run.

Reply:
Is what's described in https://github.com/algolia/shipjs/issues/986 sufficient? For now if a release is prepared, but none of the packages requires that update, shipjs will still prepare the release, possibly doing a useless release (not that big of a deal) that we can manually cancel if we don't think it's needed to release just yet, similarly as already happens from time to time today when the only changes are minor things that don't need to be released

Reply:
Got It. I created a [Jira ticket](https://algolia.atlassian.net/browse/FX-2162) so we handle this quickly, I'm not fond of such a manual process to last too long.